src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestFinder.kt

```diff
@@ -139,54 +211,96 @@ class PestFinder {
     fun onScoreboardChange(event: ScoreboardChangeEvent) {
```

Comment:
add comments above the different matchMatcher calls. its not intuitive clear what is happening at every single spot